src/wrapper.js

```diff
@@ -0,0 +1,23 @@
+(function(global, factory) {
+	if (typeof module === "object" && typeof module.exports === "object") {
+		module.exports = factory(global);
+	} else {
+		factory(global);
+	}
+	// Pass this if window is not defined yet
+}(typeof window !== "undefined" ? window : this, function(window, noGlobal) {
+
+    //@CODE
```

Comment:
I noticed, but if it's not doing anything right now, then there's no real reason to have it in the main code base, that should only exist in a branch where the "implementing what this is a placeholder for" work is happening.